contracts/validators/CredentialAtomicQueryValidatorBase.sol

```diff
@@ -9,35 +9,11 @@ import {IVerifier} from "../interfaces/IVerifier.sol";
 import {IState} from "../interfaces/IState.sol";
 import {PoseidonFacade} from "../lib/Poseidon.sol";
 
-abstract contract CredentialAtomicQueryValidator is OwnableUpgradeable, ICircuitValidator, ERC165 {
-    struct CredentialAtomicQuery {
-        uint256 schema;
-        uint256 claimPathKey;
-        uint256 operator;
-        uint256 slotIndex;
-        uint256[] value;
-        uint256 queryHash;
-        uint256[] allowedIssuers;
-        string[] circuitIds;
-        bool skipClaimRevocationCheck;
-        // 0 for inclusion in merklized credentials, 1 for non-inclusion and for non-merklized credentials
-        uint256 claimPathNotExists;
-    }
-
-    struct CommonPubSignals {
-        uint256 merklized;
-        uint256 userID;
-        uint256 issuerState;
-        uint256 circuitQueryHash;
-        uint256 requestID;
-        uint256 challenge;
-        uint256 gistRoot;
-        uint256 issuerID;
-        uint256 isRevocationChecked;
-        uint256 issuerClaimNonRevState;
-        uint256 timestamp;
-    }
-
+abstract contract CredentialAtomicQueryValidatorBase is
+    OwnableUpgradeable,
```

Comment:
As far as we anyway break backward compartibility, don't we want it to be Ownable2StepUpgradable?